File: CVR/batch-3/python/reverse.py



num = 9876

is_negative = False

if num < 0:
    is_negative = True
    num = num*-1

res = 0

while num != 0:
    rem = num%10
    res = res*10 + rem
    num = num // 10
if is_negative == True:
    print(-res)
else:
    print(res)

File: CVR/batch-4/Reverse.java
class Reverse{
    public static void main(String[] args) {
        int num = 34567 ,result = 0,temp = num ,count = 0;

        while(num !=0){
            int rem = num % 10;
            result = result * 10 + rem;
            num = num /10;
            count++;
        }

        System.out.println(result);
        System.out.println(temp % count);
    }
}